client/src/index.html

```diff
@@ -7,6 +7,10 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
```

Comment:
Is this different than the Material icons node module?

Reply:
On review, it's not. I think I jumped to conclusions about having to use that in addition to the node module. I can remove and it should be fine.